clustermesh/clustermesh.go

```diff
@@ -1776,6 +1783,7 @@ func EnableWithHelm(ctx context.Context, k8sClient *k8s.Client, params Parameter
 	helmStrValues := []string{
 		"clustermesh.useAPIServer=true",
 		fmt.Sprintf("clustermesh.apiserver.service.type=%s", params.ServiceType),
+		"externalWorkloads.enabled=true",
```

Comment:
I would suggest not to enable this unconditionally (maybe add a flag?), given that in most cases external workloads support is not needed, and it would defeat the optimization introduced in https://github.com/cilium/cilium/pull/25259.

Reply:
let's do a flag 🎏

Reply:
added a flag ✅